fastapi_sqla/__init__.py

```diff
@@ -51,38 +52,48 @@
 _Session = sessionmaker()
 
 
+def new_engine(*, envvar_prefix: str = None) -> Engine:
+    envvar_prefix = envvar_prefix if envvar_prefix else "sqlalchemy_"
+    lowercase_environ = {
+        k.lower(): v for k, v in os.environ.items() if k.lower() != "sqlalchemy_warn_20"
```

Comment:
`sqlalchemy_warn_20` quésaco?

Reply:
Part of having fastapi-sqla sqlachemy 2.0 compliant: https://github.com/dialoguemd/fastapi-sqla/pull/48

Reply:
-> https://github.com/dialoguemd/fastapi-sqla/blob/master/tests/conftest.py#L8

Reply:
-> https://docs.sqlalchemy.org/en/14/changelog/migration_20.html?highlight=sqlalchemy_warn_20#migration-to-2-0-step-two-turn-on-removedin20warnings

Reply:
Gotcha, thanks for clarifying!